### ISSUE TITLE
"contacts" entry empty

### ISSUE THREAD

**Issue by mbreitmun** [USGS-WiM/SiGLDMS#220]
I've added 5(?) projects today and I've come across this twice:

I enter "Contact" info and it doesn't show the info after saving. It shows that there is 1 contact entered (contacts.jpg). I exited out of the project and re-entered it and the contact field appears filled, but it's blank content
![contacts](https://cloud.githubusercontent.com/assets/22300964/20688805/0cf6f5d6-b588-11e6-8236-b076697a229f.JPG)
![contacts_empty](https://cloud.githubusercontent.com/assets/22300964/20688806/0cf7eb58-b588-11e6-88da-cf3b651dfa71.JPG)

 (contacts_empty.jpg)

**Comment by mbreitmun** [comment]
The problem is repeating itself. I made sure that I've clicked the "save" button after initial entry.

**Comment by troddick** [comment]
@mbreitmun , I may need you to show me this. I am not able to duplicate it. I'm thinking it may have something to do with the org/div/sec you are choosing, but I'm not sure.

**Comment by jlbruce** [comment]
Hmm. I watched Molly re-enact this, and while she couldn't get an empty Contact to appear, she did have issues when copy and pasting contact info from Excel into the DMS. It created a Contact (at least the Contact count increased), but it never showed up in the list. We were able to manually enter contacts before and after the missing copied Contact just fine.

I'm traveling now, and I've been trying to recreate this behavior on my Mac (copying from Mac Excel), and I can't reproduce it on Test. Maybe it's a PC thing? I won't be able to test it on my PC until I get back, so Tonia, if you want to try it yourself, or do hangout with Molly and watch her, feel free. Here's the process that created the error:
- Create a new project (not sure this mattered, but we were demoing in Molly's Prod account since that's where it happened).
- Add a new contact, and copy and paste from SiGL Spreadsheet V11. Be sure to copy and paste directly from the cell so you might be carrying some formatting information (not from the formula bar above, which seems to copy values as plain text).
- Select any org (we first thought it might be the Bad River Watershed Association, but it happened with other orgs).
- Save. The count on the Contacts tab should increase, but the org will not display in the list. (Previously for Molly, a white contact item box with empty headers would appear and the count would go up, but when she demoed it for me, no box appeared. Only the count went up.)
